samples/CustomReceiver.AspNetCore/project.json

```diff
@@ -0,0 +1,53 @@
+{

```

Comment:
Maybe I'm not seeing it in the commit, but where's the Program.cs or Startup.cs of this sample?